besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2210,7 +2225,8 @@ private PrivacyParameters privacyParameters(final KeyValueStorageProvider storag
       privacyParametersBuilder.setEnabled(true);
       privacyParametersBuilder.setEnclaveUrl(privacyUrl);
       privacyParametersBuilder.setMultiTenancyEnabled(isPrivacyMultiTenancyEnabled);
-      privacyParametersBuilder.setOnchainPrivacyGroupsEnabled(isFlexiblePrivacyGroupsEnabled);
+      privacyParametersBuilder.setFlexiblePrivacyGroupsEnabled(
+          isFlexiblePrivacyGroupsEnabled || isOnchainPrivacyGroupsEnabled);
```

Comment:
only thing to be careful of here is what if the user specifies both options with different values - just want to make sure it's not confusing. 
--privacy-onchain-groups-enabled=false --privacy-flexible-groups-enabled=true

Reply:
prob write a test for this, should help clarify what's happening

Reply:
50fc8f4